### PR TITLE
Create the DR template and move the existing ADRs to SIG Core

### LINKED ISSUE #55: Replace ADR template with new DR template
**Description**

- create DR template
- replace current ADR template with new DR template
- Cleanup existing ADR location and put them under SIG-Core

**Reasons**
- unify the ADRs with DRs

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Change the existing ADR template to DR
- Rename the existing ADRs and move them to the SIG core `decisions` folder
- Update the existing README.md documents to reflect the changes

**Related issue(s)**
Fixes  #55

### CODE REVIEW

**Comment by kazydek** [comment]
As agreed, we keep the non-code block format of the template at this point. To be revised in the future.

File: guidelines/templates/resources/DR.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Created on {YYYY-MM-DD} by {name and surname (@Github username)}.
+
+## Decision log
+
+| Name | Description |
+|-----------------------|------------------------------------------------------------------------------------|
+| Title | {Provide a brief summary of the decision.} |
+| Ultimate decision maker(s) | {Specify who takes the final decision if the SIG/WG cannot reach the consensus.} |
+| Due date | {Specify the date by which the SIG or WG members need to make the decision. Use the `YYYY-MM-DD` date format.} |
```
Comment (klaudiagrz) [inline guidelines/templates/resources/DR.md]: `to make a decision`

Comment (klaudiagrz) [reply guidelines/templates/resources/DR.md]: discussed 👍

**Comment by kazydek** [comment]
@Abd4llA Yes, the decision-making process will be documented as part of the #56 .

**Pull request merged**